cax/main.py

```diff
@@ -337,5 +338,377 @@ def status():
+    config.set_rucio_rse( args.rucio_rse)
+    config.set_rucio_scope( args.rucio_scope)
+    config.set_rucio_upload( args.rucio_upload )
+    
+    #This one is mandatory: hardcoded science run number!
+    config.set_rucio_campaign("000")
+
+    log_level = getattr(logging, args.log.upper())
+    if not isinstance(log_level, int):
+        raise ValueError('Invalid log level: %s' % args.log)
+
+    run_once = args.once
+    database_log = not args.disable_database_update
+
+    # Set information to update the run database 
+    config.set_database_log(database_log)
+
+    # Check passwords and API keysspecified
+    config.mongo_password()
+
+    # Setup logging
+    cax_version = 'ruciax_v%s - ' % __version__
+    logging.basicConfig(filename=args.logfile,
+                        level=log_level,
+                        format=cax_version + '%(asctime)s [%(levelname)s] '
+                                             '%(message)s')
+    logging.info('Daemon is starting')
+    logging.info('Logfile: %s', args.logfile)
+    
+    # define a Handler which writes INFO messages or higher to the sys.stderr
+    console = logging.StreamHandler()
+    console.setLevel(log_level)
+
+    # set a format which is simpler for console use
+
+    formatter = logging.Formatter('%(name)-12s: %(levelname)-8s %(message)s')
+    # tell the handler to use this format
+    console.setFormatter(formatter)
+    # add the handler to the root logger
+    logging.getLogger('').addHandler(console)
+
+    # Get specified cax.json configuration file for cax:
+    if args.config_file:
+        if not os.path.isfile(args.config_file):
+            logging.error("Config file %s not found", args.config_file)
+        else:
+            logging.info("Using custom config file: %s",
+                         args.config_file)
+            config.set_json(args.config_file)
+
+    tasks = [
+        data_mover.CopyPull(),
+        data_mover.CopyPush(),
+        rucio_mover.RucioRule()
+    ]
+
+    # Raises exception if unknown host
+    config.get_config()
+
+    user_tasks = config.get_task_list()
+
+    while True:
+        for task in tasks:
+            name = task.__class__.__name__
+            
+            # Skip tasks that user did not specify
+            if user_tasks and name not in user_tasks:
+                continue
+            
+            logging.info("Executing %s." % name)
+
+            try:
+              if args.name is not None:
+                task.go(args.name)
+              else:
+                task.go(args.run)
+                
+            except Exception as e:
+                logging.fatal("Exception caught from task %s" % name,
+                              exc_info=True)
+                logging.exception(e)
+                raise
+
+        # Decide to continue or not
+        if run_once:
+            break
+        else:
+            logging.info('Sleeping.')
+            time.sleep(60)
+
+def massiveruciax():
```

Comment:
massiveruciax() and massive() have not necessarily something in common:
* massive() submits jobs for processing or file copy to Tegner or midway queue.
* massiveruciax() and also massivetsm() submit jobs in a sequence and do not involve any workload manager.